src/pages/Main/Search/Histogram.tsx

```diff
@@ -41,9 +54,7 @@ const CachedHistogramView = React.memo(function HistogramView({ query }) {
         return {
             labels: data.map(
                 (d) =>
-                    `${d.month}.${(parseInt(d.year) % 100)
```

Comment:
why did you get rid of the parseInt?

Reply:
parseInt is unnecessary because the year is already a number.